x-pack/test/upgrade_assistant_integration/upgrade_assistant/reindexing.js

```diff
@@ -123,7 +123,10 @@ export default function ({ getService }) {
 
     it('shows no warnings', async () => {
       const resp = await supertest.get(`/api/upgrade_assistant/reindex/7.0-data`);
-      expect(resp.body.warnings.length).to.be(0);
+      // By default all reindexing operations will replace an index with an alias (with the same name)
```

Comment:
I wonder if this `shows no warnings` test shouldn't be also in `7.17`?

Reply:
Yeah I guess we could have a similar test there, I'll see if I can add it in a subsequent PR!